yarn-project/pxe/src/note_processor/note_processor.ts

```diff
@@ -255,7 +255,7 @@ export class NoteProcessor {
     const noteDaos: NoteDao[] = [];
     for (const deferredNote of deferredNoteDaos) {
       const { note, contractAddress, storageSlot, txHash, newCommitments, dataStartIndexForTx } = deferredNote;
-      const payload = new L1NotePayload(note, contractAddress, storageSlot);
+      const payload = new L1NotePayload(note, contractAddress, storageSlot, new Fr(42)); // TODO(nventuro): fix DeferredNoteType
```

Comment:
Is this TODO still relevant? If yes and you don't want to tackle it in this PR then please create an issue for this so that we don't forget and link it here (e.g. TODO(#5672)).

Reply:
No, I had missed this :grimacing: